### PR TITLE
Make Tieflings Latino again (Tiefling Accent return + tweaks)

### PR DESCRIPTION
**Pull request by Stokeling9701**
## About The Pull Request

Do you like the drow accent? (maybe not)
Have you liked the idea of racially profiling someone based off of their accent? (uhm ofc u have)
Did you feel DISSAPOINTMENT when you realized tiefling have a vaquero role, but not a vaquero accent or way of speaking?

THEN THIS IS THE PR FOR YOU

This simply adds a new spanish_replacement.json file (spanish accent) that will give tieflings back their latino coded voice!
now youll be beheaded for calling the queen senora! how wicked!

Tieflings are SUPPOSED to be shunned for being different, now you cant hide under your cloak since youre easier to tell apart from voice alone!.

TLDR; Adds tiefling accent back, with new terms.

## Why It's Good For The Game

why is this needed?? WHY IS THIS NEEDED?? its simple really.

Tieflings are TOO CUTE, theyre not hateable enough. For a shunned race, I dont feel like theyre different enough to warrant being shunned when theres draconids, kobolds, and half orcs running around looking either super crazy, or having a crazy accent.

With an accent, some people will understandably not like having a forced accent now. However drow have dealt with it for far longer, and their accent enhances the rp around discriminating them. Whats a better way to make people hate you then a thick accent? I tried to not make it too overwhelming, it has some replacer words but my intention was to have a similar experience to dealing with drow as they both have a line count of around 45 or so.

In addition, having this accent also strengthens their lore without having to write a single drop OF lore. The concept of tieflings being these Vaqueros who come from far away lands and bring their language and culture with them the same way drow do is just extremely endearing. I urge that any opponents of this dont completely write off the idea of a tiefling accent.

I know theres a PR for a new accent system, this is just a bandaid till something like that gets added since we have to make due with the current accent system.

TLDR; its good because tieflings are a shunned race that should be more different then just horns and a tail, when half the races HAVE horns and a tail, an accent makes it easier to shun them beyond "you look like devil".

### CODE REVIEW

**Comment by X404ErrorX** [comment]
A small correction, "Thats lovely" is not "Que rico", it should be "Que adorable" or "Adorable", the meaning of "Que rico" is something about the taste, a meal can be "Rica" or someone with a lot of money Rico/Rica (rich), but it have nothing to do with "lovely".

**Comment by Stokeling9701** [comment]
> A small correction, "Thats lovely" is not "Que rico", it should be "Que adorable" or "Adorable", the meaning of "Que rico" is something about the taste, a meal can be "Rica" or someone with a lot of money Rico/Rica (rich), but it have nothing to do with "lovely".

im not gonna lie my whole point of "thats lovely" -> being "que rico" is entirely for a freaky context, typically in sex which is when youd say that.

**Comment by Alanii** [comment]
Tieflings were Spanish. 

South America in this world is still inhabited by heart cutting out tribal emperors most likely.

**Comment by TheFenix323** [comment]
I say, this is barely accent, they are just speaking spanish at this point.

**Comment by Stokeling9701** [comment]
> I say, this is barely accent, they are just speaking spanish at this point.

its meant to make them  different in a similar way to orc or drow, just to make them easier to hate

**Comment by Useroth** [comment]
I think we won't be making the tieflings latino
